1-js/08-prototypes/01-prototype-inheritance/4-hamster-proto/solution.md

```diff
@@ -20,8 +20,8 @@ let hamster = {
 
   eat(food) {
 *!*
-    // assign to this.stomach instead of this.stomach.push
-    this.stomach = [food];
+    // conditional assignment
+    this.stomach ? this.stomach = [food] : this.stomach.push(food);
```

Comment:
This won't work. `this.stomach.push(food)` won't be executed because property `stomach` is in the prototype chain. You have to use `hasOwnProperty` method instead.

```
this.hasOwnProperty('stomach') ? this.stomach.push(food) : this.stomach = [food];
```